grails-forge-core/src/main/java/org/grails/forge/feature/github/workflows/plain/templates/plainGithubWorkflow.rocker.raw

```diff
@@ -12,23 +12,24 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions:
+      checks: write
+      contents: read
```

Comment:
These permissions are needed for the `scacap/action-surefire-report` action to work properly

Reply:
This can be removed if we use build scans instead (see below).

Reply:
> These permissions are needed for the scacap/action-surefire-report action to work properly

Are you sure the permissions block is needed?
The example at https://github.com/ScaCap/action-surefire-report?tab=readme-ov-file#example-usage states that
> \# Permissions block is optional, useful for dependabot checks